libs/pdi-scanner/src/scanner.rs

```diff
@@ -219,7 +213,10 @@ impl Scanner {
     /// dropped.
     pub fn stop(&mut self) {
         if let Some(stop_tx) = self.stop_tx.take() {
-            stop_tx.send(()).unwrap();
+            // send will only error if its receiver has been dropped, which
+            // means the scanner is already cleaned up, so it's safe to ignore
+            // the error here.
+            stop_tx.send(()).ok();
```

Comment:
From what I've seen, the idiomatic way to ignore a result is like so:
```suggestion
            let _ = stop_tx.send(());
```